src/main/target/CJMCU/initialisation.c

```diff
@@ -31,6 +31,6 @@ void targetBusInit(void)
 
     if (!doesConfigurationUsePort(SERIAL_PORT_USART3)) {
         serialRemovePort(SERIAL_PORT_USART3);
-        i2cInit(I2C_DEVICE);
+        i2cInitAll();
```

Comment:
Call    `i2cInitAll(i2cPinConfig());` - will need to include `include "config/config_master.h"`